apps/studio/src/pages/home/wallet/Portfolio.tsx

```diff
@@ -52,27 +53,31 @@ const Portfolio: FunctionComponent = () => {
     setRowsPerPage(rowsToRender);
   }, [windowHeight]);
 
+  if (isLoading) {
+    return (
+      <TableSkeleton
+        cols={ windowWidth && windowWidth > theme.breakpoints.values.sm ? 3 : 2 }
+        maxWidth={ maxListWidth }
+        rows={ skeletonRows }
+      />
+    );
+  }
+
+  if (!isLoading && !songs.length) {
+    return <EmptyPortfolio />;
+  }
```

Comment:
I noticed, even though I have songs in my portfolio, navigating between transaction tab and portfolio tab, the `EmptyPortfolio` component displays first for a second and then the skeleton appears and then loads my portfolio. Should probably go straight to skeleton and then to portfolio.